superchain/configs/sepolia-seal-0/superchain.yaml

```diff
@@ -0,0 +1,11 @@
+name: Sepolia Seal 0
+l1:
+  chain_id: 11155111
+  public_rpc: https://eth-sepolia.g.alchemy.com/v2/9PqDKhFhwoeNRoRLkSGoZ7Kz6hTdTwv6
+  explorer: https://sepolia.etherscan.io
+
+# todo: `protocol_versions_addr` was just copy-pasted from the sepolia-dev-0 one cause conduit
```

Comment:
okay I see, I think there's just one canonical protocol_versions_addr per network, so that makes sense

Reply:
There is one per superchain target, not one per L1 network. I don't think we want to copy-paste, here.

An alternative would be to add `sepolia-seal-0` as a new chain with the existing `sepolia` or `sepolia-dev-0` superchain target set.

Was launching a new superchain ("set of chains) , as well as a new rollup chain intended?

Reply:
the intention was to keep the config separate from the main sepolia superchain configs in case there were side effects we're not aware of. will add a protocol version specific to this target